### PR TITLE
Using the new back-end visit operations to help determine current date on back-end

### PR DESCRIPTION
**Pull request by edrisse**
Requires https://github.com/esaude/openmrs-module-poc/pull/11

### CODE REVIEW

**Comment by edrisse** [comment]
[Let back-end determine creation date](https://trello.com/c/ZCKAhOtb/66-let-back-end-determine-creation-date)

File: app/poc-common/clinical-services/service-form/components/formWizard.js [hunk 1]
```diff
@@ -239,19 +239,7 @@
     }
 
     function checkIn() {
-      var visitType = _.find($rootScope.defaultVisitTypes, function (o) {
```
Comment (ynurmahomed) [inline app/poc-common/clinical-services/service-form/components/formWizard.js]: @edrisse there are different visitTypes for `clinic` and `pharmacy`, hope this is handled.

Comment (edrisse) [reply app/poc-common/clinical-services/service-form/components/formWizard.js]: Hum, will take a look at that. Don't merge yet.

Comment (edrisse) [reply app/poc-common/clinical-services/service-form/components/formWizard.js]: @Fixed on the back-end. You can proceed with the review.

**Pull request merged**